modules/dns/README.md

```diff
@@ -31,6 +31,33 @@ The returned `flags` parameter means that when you run a `terraform plan` comman
             # (12 unchanged attributes hidden)
         }
 
+The DNS records also can be imported using a file in IBM Cloud Internet Services. This module allows the import of DNS records in two ways:
+- Directly importing the `dns_record.txt` in BIND format.
+- Do base64 encoding on the `dns_record.txt` in BIND format and pass it as a string.
+
+For converting the dns_record text file to Base64 encoding.
+```sh
+cat dns_records.txt | base64
```

Comment:
The DNS records can be imported from a file in IBM Cloud Internet Services. This module allows the import of DNS records in two ways:
- using the local file path (if accessible)
- using the base64 encoded string for the file

To convert the records text file to base64 encoded string:

```sh
cat dns_records.txt | base64

Reply:
Made the changes.